pyseir/deployment/webui_data_adaptor_v1.py

```diff
@@ -123,13 +123,18 @@ def map_fips(self, fips):
         all_hospitalized_today = None
         try:
             fit_results = load_inference_result(fips)
-        except ValueError:
+        except (KeyError, ValueError):
             fit_results = None
-            logging.error(f'Fit result not found for {fips}: Skipping inference elements')
+            logging.warning(f'Fit result not found for {fips}: Skipping inference elements')
 
         for i_policy, suppression_policy in enumerate(policies):
             if suppression_policy == 'suppression_policy__full_containment':  # No longer shipping this.
                 continue
+            if suppression_policy == 'suppression_policy__inferred' \
+                    and len(fips) == 5 \
```

Comment:
what is this for?

Reply:
I think it's so we don't try to apply the whitelist to states (which have fips codes of length 2)